<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn</class>
 <widget class="QDialog" name="SignIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="AccountEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0px;
border-radius: 16px;
background-color: white;
padding-left: 12px;
font: 15pt &quot;Myriad Pro Cond&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Tên tài khoản</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0px;
border-radius: 16px;
background-color: white;
padding-left: 12px;
font: 15pt &quot;Myriad Pro Cond&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Mật khẩu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogInButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#LogInButton 
{
	border-width: 0px;
	border-radius: 16px;
	font: 15pt &quot;Myriad Pro Cond&quot;;
	background: #00ff80;
	color: #555;
	outline: none;
	font-weight: bold;
}
#LogInButton:hover 
{
	background-color: #66ffb2;
	color: #000;
}</string>
   </property>
   <property name="text">
    <string>Đăng nhập</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ResultLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 63 12pt &quot;Myriad Pro Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="RoleComboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#RoleComboBox 
{
	border: 1px solid darkgray;
	border-radius: 16px;
	font: 15pt &quot;Myriad Pro Cond&quot;;
	color: #222;
	outline: none;
	padding-left: 20px;
	font-weight: bold;
}
#RoleComboBox:!editable, #RoleComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #eee, stop: 0.4 #ddd,
                                stop: 0.5 #d8d8d8, stop: 1.0 #ddd);
}
#RoleComboBox:!editable:on, #RoleComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #e9e9e9, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #e9e9e9);
	outline: none;
}

#RoleComboBox:on {
    padding-top: 3px;
    padding-left: 20px;
}

#RoleComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
   	border-left: 2px solid darkgray;
}
#RoleComboBox::down-arrow {
    image: url(images/drop_down.png);
}

#RoleComboBox::down-arrow:on { 
    top: 1px;
}

QComboBox QAbstractItemView
{
    border: 1px solid darkgray;
    color: black;
    selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #111, stop: 1 #555);
	background: white;
	outline: none;
	padding-left: 20px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Vai trò...</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Độc giả</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thủ thư</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
